extensions/amp-form/0.1/validation-bubble.js

```diff
@@ -103,7 +135,13 @@ function measureTargetElement(state) {
  * @private
  */
 function showBubbleElement(state) {
-  state.bubbleElement.textContent = state.message;
+  removeChildren(state.bubbleElement);
+  const messageDiv = state.bubbleElement.ownerDocument.createElement('div');
+  messageDiv.id = state.bubbleElement.getAttribute('aria-labeledby');
+  messageDiv.textContent = state.message;
+  state.bubbleElement.setAttribute('role', 'alert');
```

Comment:
Do you have a reference to the actual input causing the validation error? If so, giving this an id and setting `aria-describedby` on that input would be useful ( if it already has `aria-describedby` set, can add this id to it using space as separator, and of course removing it in hideBubble)